creator-node/src/hashids.js

```diff
@@ -0,0 +1,19 @@
+const Hashids = require('hashids/cjs')
+
+const HASH_SALT = "azowernasdfoia"
+const hashids = new Hashids(HASH_SALT, 5)
```

Comment:
can you throw some quick comments for these two lines / maybe link to pkg docs 🙏

Reply:
Will do